### ISSUE TITLE
Defer JS Until After Pageload

### ISSUE THREAD

**Issue by buschschwick** [factor1/factor1_blankwp#4]
```php
// Defer Javascripts
// Defer jQuery Parsing using the HTML5 defer property
if (!(is_admin() )) {
    function defer_parsing_of_js ( $url ) {
        if ( FALSE === strpos( $url, '.js' ) ) return $url;
        if ( strpos( $url, 'jquery.js' ) ) return $url;
        // return "$url' defer ";
        return "$url' defer onload='";
    }
    add_filter( 'clean_url', 'defer_parsing_of_js', 11, 1 );
}
```

Adding this defers parsing of all JS until after pageload. Goes in `functions.php`.

**Comment by buschschwick** [comment]
Important note, we need to point to the jquery we are pulling in or else it will conflict. For instance, this is a correct working version from Health Corps. 

```php
// Defer Javascripts
// Defer jQuery Parsing using the HTML5 defer property
if (!(is_admin() )) {
    function defer_parsing_of_js ( $url ) {
        if ( FALSE === strpos( $url, '.js' ) ) return $url;
        if ( strpos( $url, 'jquery.min.js' ) ) return $url;
        // return "$url' defer ";
        return "$url' defer onload='";
    }
    add_filter( 'clean_url', 'defer_parsing_of_js', 11, 1 );
}
	
// Load jQuery
	if ( !is_admin() ) {
	   wp_deregister_script('jquery');
	   wp_register_script('jquery', ("http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"), false);
	   wp_enqueue_script('jquery');
	}
```

Note the jquery.min.js file in the defer function is the same as the jquery.min.js we are registering/enqueuing.

**Comment by buschschwick** [comment]
Implemented in 2.0.0
